Enterprise-CMCS/macpro-onemac
# OY2 24628: OneMAC Package Details - Reverse Chronological order

Story: https://qmacbis.atlassian.net/browse/OY2-24628
Endpoint: See github-actions bot comment

### Details
A reconfiguration of package details pages.

### Changes
- removed a bunch of extraneous, leftover, console.log statements
- consolidated all OneMAC events (except for Admin Changes) into the reverseChrono attribute
- mapped which component types map to which event types
- created the packageAttributes configuration attribute to tell the builder which attributes get promoted to the package level.
- added a workaround for withdraw package logic
- configured the details page to follow the Fakema
- updated seed data

### Implementation Notes
- this work will require a complete rebuild of all packages
- withdraw package event currently uses currentStatus: "Submitted" and should follow the new pattern of containing the PACKAGEs new status instead of the status of the event submission.

### Test Plan
NOTE: THIS IS REALLY DIFFICULT to test without the automated testing... so I am putting this up for review to get a "reality check" on if this is the right direction and am hoping that updating the tests will uncover any issues that I missed.

1. Login as a OneMAC user
2. navigate to the details pages of different package types in different statuses
3. Verify the reverse chrono format contains the history in the expandable sections, with the most recent event expanded
4. Verify the Admin Changes (if any) are below the reverse chronological areas.
